### DISCUSSION TITLE
How do I publish the wizard view?

### DISCUSSION THREAD

**Discussion by pekka** [filamentphp/filament#7057] [Help] [answered]
I need to publish the Wizard's view

`/vendor/filament/forms/resources/views/components/wizard.blade.php`

(because I need to style the "next" button; I know publishing views is generally discouraged, but I see no other option here!)

![image](https://github.com/filamentphp/filament/assets/262464/48756126-500a-488c-97f3-03dc95007aac)

But can't figure out how!
I've tried placing it in

```
/resources/views/filament/forms/resources/views/components/wizard.blade.php
/resources/views/filament/components/wizard.blade.php
/resources/views/components/forms/wizard.blade.php
/resources/views/components/wizard.blade.php
```

But to no avail.

Which is the correct place?

**Comment by danharrin** [comment]
Can you not use CSS to style the button?

**Reply by pekka** [reply]
Not seeing a class or property I could use to target the specific button! 

![image](https://github.com/filamentphp/filament/assets/262464/b223d4eb-a864-4f08-b2db-75b18d84f001)

**Reply by pboivin** [reply]
Maybe this?
```css
.filament-forms-wizard-component button {
  /* ... */
}
```

**Reply by pboivin** [reply]
Ah, I see what you mean... it's not easy to style only the Prev/Next buttons.

**Reply by pekka** [reply]
Yeah, I'm sure one could cook up something hacky using `nth-of-type` or something but then it seems easier to publish the entire view. Happy to submit a PR to give those elements distinctive classes but with v3 around the corner I'm not sure it's worth the effort!

**Comment by danharrin** [comment, marked as answer]
In v3 you can do this using `->nextAction(fn ($action) => $action->view('...'))`

**Reply by pekka** [reply]
In that case, happy to wait! Thanks!
